docs/on-premises/single-node/002-single-node-onprem-install.md

```diff
@@ -230,17 +230,29 @@ kubectl -n flyte port-forward service/flyteconsole 8088:80
 ## Using Flyte
 
 4. Save the following "hello world" workflow definition:
+Check that the onprem-flyte-binary-values.yaml has these values as it will result in a 10x faster execution for this simple example.
+
+    task_resources:
+      defaults:
+        cpu: 1000m
+        memory: 2Gi #change default requested resources and limits to fit your needs
+      limits:
+        cpu: 3000m
+        memory: 4Gi
 
 ```bash
 cat <<<EOF >hello_world.py
-from flytekit import task, workflow
-@task
+from flytekit import Resources, task, workflow
+
+@task(requests=Resources(cpu="900m", mem="3Gi"))
```

Comment:
I haven't been able to reproduce this. I mean, using the same numbers increases execution time instead of dramatically reducing like in your case.
It doesn't hurt much (unless you don't have available the resources requested here) so we can leave it like you propose.